### PR TITLE
Fix/tests and fixes

### PR DESCRIPTION
**Pull request by nytamin**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
This PR adds a few tests for functions in `meteor/lib/`

* **Other information**:

**Status**

- [ ] The functionality has been tested by the PR author
- [ ] The functionality has been tested by NRK

### CODE REVIEW

**Comment by jstarpl** [comment]
There seem to be some issues with the tests - time is off by 1H, perhaps it's a timezone thing somehow?

File: meteor/lib/__tests__/lib.test.ts [hunk 1]
```diff
@@ -398,4 +402,79 @@ describe('lib/lib', () => {
...
+			c: {
+				d: 1
+			},
+			e: null,
+			f: undefined,
+		}
+		expect(partialExceptId(o)).toEqual(o) // The function only affects typings
+	})
+	testInFiber('formatDateTime', () => {
+		expect(formatDateTime(1578295344070)).toBe('2020-01-06 08:22:24')
```
Comment (jstarpl) [inline meteor/lib/__tests__/lib.test.ts]: This test doesn't pass on CircleCI, perhaps it's a problem with the Timezone?

Comment (nytamin) [reply meteor/lib/__tests__/lib.test.ts]: True, I'll look into it :)

Comment (nytamin) [reply meteor/lib/__tests__/lib.test.ts]: Fixed now, I ended up having to skip the specific tests when running locally on Windows, as there seems to be a bug in how the timezone settings are applied in Node on Windows. ref: https://www.npmjs.com/package/set-tz#windows-caveat

**Pull request merged**